src/tests/tests.h

```diff
@@ -623,15 +641,20 @@ template<typename Test_Class>
 class TestClassRegistration
    {
    public:
-      TestClassRegistration(const std::string& category, const std::string& name)
+      TestClassRegistration(std::string category, std::string name, CodeLocation registration_location)
```

Comment:
What is the reason behind taking by value if `Test::register_test()` is still taking them by reference? Same in `TestFnRegistration`.

Reply:
Nothing particular. I wanted to modernize this but didn't follow through all the way. Didn't remove those changes in the end, though.